test/bdd/agent_controller_steps.go

```diff
@@ -37,21 +38,24 @@ const (
 // AgentWithControllerSteps
 // TODO all steps in this package needs to be placed in different packages instead of sharing same package [Issue #584]
 type AgentWithControllerSteps struct {
-	bddContext     *Context
-	controllerURLs map[string]string
-	webhookURLs    map[string]string
-	invitations    map[string]*didexchange.Invitation
-	connectionIDs  map[string]string
+	bddContext          *Context
+	controllerURLs      map[string]string
+	webhookURLs         map[string]string
+	invitations         map[string]*didexchange.Invitation
```

Comment:
invitations already exists in context

Reply:
@sudeshrshetty  ^^^^

Reply:
Removed [context](https://github.com/hyperledger/aries-framework-go/compare/74c367d5b7205397949e492af585a2932933e9bf..2f9b5a131019043c90d08d749ac76558f8a77e52#diff-a2605ce83886c78952f7caa29930a5ceL41).